protocol/0067-KEYS-key_management.md

```diff
@@ -243,16 +243,41 @@ need to be hashed individually).
 
 ## Acceptance Criteria 
 
-### Ethereum key - multisig (<a name="0067-KEYS-001" href="#0067-KEYS-001">0067-KEYS-001</a>)
-
-1. A Vega network is running with 3 validators, `v1,v2,v3` with Ethereum keys `k1, k2, k3_old`; each with equal tendermint and multisig weight.
-2. Validator `v3` has ethereum multisig public key `k3_old`. They submit a transaction to replace by ethereum multisig public key `k3_new`.
-3. The network issues a signature bundle to update that can be submitted to the ethereum multisig contract to update the key there. 
-4. This is submitted to ethereum; the multisig contract is updated.
-5. Vega nodes receive the event confirming the key has been updated. 
-6. Party `p` now issues a withdrawal transaction. A withdrawal bundle is created utilizing `k1,k2,k3_new`. 
-7. Party `p` submits the withdrawal bundle to ethereum; multisig contract accepts it and transfers the funds on the ethereum chain. 
-
+### Generic 
+
+1. Rewards are produced post all key rotations (<a name="0067-KEYS-006" href="#0067-KEYS-006">0067-KEYS-006</a>)
+1. After a key rotation the node still has the ability to self stake/delegate and delegate to other validator nodes (<a name="0067-KEYS-007" href="#0067-KEYS-007">0067-KEYS-007</a>)
```

Comment:
Can these clarify which key is being rotated? I'm assuming its the vega key and not the ethereum key?